runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -56,37 +67,45 @@ public class JwtGuardHandler implements GuardHandler
     public JwtGuardHandler(
         JwtOptionsConfig options,
         LongSupplier supplyAuthorizedId)
+    {
+        this(options, supplyAuthorizedId, url -> "{}");
+    }
```

Comment:
This constructor seems to be used only by unit tests, let's remove it and supply the last parameter from the unit test instead.